asmcomp/arm64/emit.mlp

```diff
@@ -1273,9 +1273,7 @@ let end_assembly () =
       efa_string = (fun s -> emit_string_directive "	.asciz	" s) };
   emit_symbol_type emit_symbol lbl "object";
   emit_symbol_size lbl;
-  begin match Config.system with
-  | "linux" ->
-      (* Mark stack as non-executable *)
-      `	.section	.note.GNU-stack,\"\",%progbits\n`
-  | _ -> ()
+  if Config.nonexecstack_note then begin
+    (* Mark stack as non-executable *)
+    `	.section .note.GNU-stack,\"\",%progbits\n`
```

Comment:
Maybe we could have a `emit_nonexexstack_note` helper in Emitaux to hide the Config check here?

Reply:
I just waited for an explicit suggestion to do so :-)

Reply:
Done.